compose/runtime/runtime/src/wasmTest/kotlin/kotlinx/test/IgnoreTargetsActuals.wasm.kt

```diff
@@ -14,19 +14,8 @@
  * limitations under the License.
  */
 
-package androidx.compose.ui.graphics
-
-import org.khronos.webgl.ArrayBuffer
-import org.khronos.webgl.ArrayBufferView
-import org.khronos.webgl.Int32Array
-import org.khronos.webgl.Uint8Array
-import org.khronos.webgl.get
-
-internal actual fun ByteArray.putBytesInto(array: IntArray, offset: Int, length: Int): Unit {
-   val byteBuffer = Uint8Array(this.toTypedArray()).buffer
-   val intArray = Int32Array(byteBuffer, offset, length)
-   for (i in 0 until intArray.length) {
-      array[i] = intArray[i]
-   }
-}
+package kotlinx.test
```

Comment:
I can't understand why it considers this file as a moved one. It's a new file.

Reply:
weird. IDEA thinks it's moved too

Reply:
deleted the file and pushed it again. Seems to be fixed now.

Reply:
sometimes IDEA uses its own logic of detecting moves, so this is ok